docs/accessibility_guidelines.md

```diff
@@ -0,0 +1,73 @@
+
+To define and make clearer accessibility scenarios, [W3C](https://www.w3.org/) created a standard called Web Content Accessibility Guidelines or shorter [WCAG](https://www.w3.org/TR/WCAG21/).
+
+As the name says, this standard was created for the web (initially developed in the 90s and ‘00s). At that time, mobile devices were not primarily used as they are today. Because of that, WCAG had some updates and changes over the years, but the most important thing this standard has defined are principles for covering cases for application accessibility.
+
+There are four main principles:
+
+* [Perceivable](principles/perceivable_principle.md)
+* [Operable](principles/operable_principle.md)
+* [Understandable](principles/understandable_principle.md)
+* [Robust](principles/robust_principle.md)
+
+## 1. Perceivable
+
+This principle covers the application content and how users can access every part of the application.
+
+## 2. Operable
+
+The [operable chapter in the WCAG standard](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview&levels=aa%2Caaa&technologies=smil%2Cpdf%2Cflash%2Csl#principle2) describes how disabilities might make it hard for some users to interact with the app. It contains information about how they alter the way users perform in-app actions and how to ensure users with nonstandard input devices can operate all parts of the app.
+
+#### Keyboard Accessible
+
+The app should be fully functional for users whose only input device is the keyboard. Operating via the keyboard should be concise and possible without requiring specific timing for individual keystrokes. Navigating into and out of different parts of the app should be possible without getting stuck. Document unconventional keyboard navigation methods. For keyboard shortcuts that only require pressing a single letter or number, the user should be able to turn the shortcut off or remap it.
+
+#### Enough Time
+
+Some users might have trouble grasping and interacting with the content that is available only for a limited time or is auto-updating. If possible, there should be a way to extend, pause or disable the timing.
+
+#### Seizures and Physical Reactions
+
+For some users, excessive flashing might cause seizures. As a general rule, avoid content that flashes more than three times in any one-second period.
+
+#### Navigable
+
+The app's navigation structure should be concise and easy to understand. Users should have an indication of where they are located and how their actions will affect their location. There should be a way to bypass content that is repeated on multiple screens. The focus order should be clear and predictable.
+
+#### Input Modalities
+
+Some users might have trouble moving the device themselves or performing complex gestures. Strive for the app to be operable using a variety of input methods. If possible, the user should have a way to undo the interaction and disable the input methods he might trigger accidentally.
```

Comment:
I believe this part can go into document about the principles itself (where the definition of every point of the category is described). This way, the reader should refer to this file before reading that one. E.g. I would put that on top of the `operable_principle.md` (first chapter) where the category can be described.

And this fille would be short and descriptive (e..g what does contains - earlier version).